packages/dbos-cloud/databases/databases.ts

```diff
@@ -451,15 +450,11 @@ export async function connect(
     console.log(`Database Username: ${databaseUsername}`);
     console.log(`Status: ${userDBInfo.Status}`);
 
-    logger.info(`Loading cloud database connection information into ${dbosConfigFilePath}...`);
-    const config: ConfigFile = loadConfigFile(dbosConfigFilePath);
-    config.database.hostname = userDBInfo.HostName;
-    config.database.port = userDBInfo.Port;
-    config.database.username = databaseUsername;
-    config.database.password = password;
-    config.database.local_suffix = local_suffix;
-    writeConfigFile(config, dbosConfigFilePath);
-    logger.info(`Cloud database connection information loaded into ${dbosConfigFilePath}`);
+    const displayPassword = showPassword ? password : password.replace(/./g, '*');
```

Comment:
Connect should only print the connection string, not the other stuff it used to print, and it should only prompt for a password if you asked to display it (otherwise it's for nothing).

Reply:
fixed

![Screenshot 2025-04-04 at 13 17 03](https://github.com/user-attachments/assets/21ee56da-9c39-45f3-921d-1734813b4aa5)